packages/wallet-sdk/global.d.ts

```diff
@@ -0,0 +1,8 @@
+export {};
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toThrowEIPError(code: number, message: string): R;
+    }
+  }
+}
```

Comment:
do we need this even thought it is only for tests? @cb-jake

Reply:
Yeah I think so. We're getting type errors in the test and our tests should be type safe as well

Reply:
gotcha. i think i added this to jest.setup.ts and it's been working so far
do we need to update anything on it?

Reply:
@cb-jake it might be because `jest.setup.ts` got missed on tsconfig? 
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/f17efb39-b36a-416b-9ce3-9f1eba1026ca)

Reply:
Oh maybe it did get missed let me try that real quick

Reply:
@bangtoven Looks like including the setup file fixed the types! New changes are pushed now